spec/polymorphism.spec.ts

```diff
@@ -134,4 +134,106 @@ describe('Polymorphism', () => {
       },
     });
   });
+
+  it('correctly creates references to existing schema objects using refId', () => {
```

Comment:
We already have multiple tests for that. You can check:
* it('creates separate schemas and links them' ...
* it('supports records with refs' ...
* it('supports nullable for registered schemas' ...
So I don't think this test verifies anything new, does it?